### DISCUSSION TITLE
How to embedded a Font Awesome Icon inside a Link

### DISCUSSION THREAD

**Discussion by arthurborgesdev** [vercel/next.js#14606] [Help] [answered]
Hello! I'm trying to make a Fontawesome Icon clickable, and when it gets clicked, to load another page of /pages folder. 

This is my code:
```
<section>
  <Link>
    <a href="/counter"><FontAwesomeIcon icon={faChevronRight} /></a>
  </Link>
</section>
``` 

But I'm getting these errors:
![errors](https://user-images.githubusercontent.com/38326017/85904181-77473700-b7de-11ea-9136-4bf974b2bb64.png)

What I'm doing wrong?

**Comment by arthurborgesdev** [comment, marked as answer]
I was using Link component totally wrong! Here is the correct code:

```
<section>
  <Link href="/counter">
    <a><FontAwesomeIcon icon={faChevronRight} /></a>
  </Link>
</section>
```

**Reply by i3130002** [reply]
As in [here](https://nextjs.org/docs/pages/api-reference/components/link#version-history) 

Version: v13.0.0  Changes: No longer requires a child <a> tag.
